charts/spdk-csi/latest/spdk-csi/templates/node.yaml

```diff
@@ -147,4 +147,4 @@ spec:
           name: simplyblock-csi-cm
       - name: csi-secret
         secret:
-          secretName: simplyblock-csi-secret
+          secretName: simplyblock-csi-secret-v2
```

Comment:
why change the secretName? we used this secretName many places like here 
- https://github.com/simplyblock-io/simplyblock-csi/blob/master/charts/sb-controller/latest/templates/storage-node-controller.yaml#L34C21-L34C43

- https://github.com/simplyblock-io/sbcli/blob/main/simplyblock_web/templates/storage_deploy_spdk.yaml.j2#L113

Reply:
I am creating a new secret in charts/spdk-csi/latest/spdk-csi/templates/secret.yaml and using that new secret. Updating the existing secret is breaking the existing storage node add node. Hence created a new secret.